.eslintrc

```diff
@@ -29,6 +29,7 @@
   },
   "rules": {
     "backpack/use-tokens": 2,
+    "react/destructuring-assignment": 0,
```

Comment:
This one could be controversial (and in particular involves making hundreds of changes to our codebase). I'm opting to tackle it in a follow up PR / decision.